sopel/builtins/units.py

```diff
@@ -199,3 +209,5 @@ def mass(bot, trigger):
         stupid_part = '{:.2f} oz'.format(ounce)
 
     bot.say('{} = {}'.format(metric_part, stupid_part))
+
+    return None
```

Comment:
Same here: I probably would have left off the explicit return

Reply:
I thought [my comment on the other thread](https://github.com/sopel-irc/sopel/pull/2616#discussion_r1801069390) would cover this, but it's the same here. The function returns `int | None`, so it must _always_ explicitly return a value.

Personally I don't mind the implicit `return None` at the end of a function, but the type-checker clearly does. 😸